packages/fxa-payments-server/src/components/Survey/README.md

```diff
@@ -0,0 +1,30 @@
+### Example Usage
+
+```javascript
+
+  import Survey, { CreateHandleIframeTask } from '@fxa-components/Survey';
+
+  const [showSurvey, setShowSurvey] = useState(true);
+  const [surveyComplete, setSurveyComplete] = useState(false);
+
+  const handleIframeTask = CreateHandleIframeTask(() => {
+    setSurveyComplete(true);
+    setTimeout(() => {
+      setShowSurvey(false);
+    }, 300);
+  });
+
+  useEffect(() => {
+    window.addEventListener('message', handleIframeTask);
+    return () => window.removeEventListener('message', handleIframeTask);
+  });
+
+  // ...
+
+  return (
+    <div className='my-app'>
+      {showSurvey && <Survey surveyUrl={'https://my-survey-url.com'} surveyComplete={surveyComplete}>}
```

Comment:
The syntax snob in me has to point out the `{...{surveyComplete}}` sugar.

Reply:
😂 Lauren I saw the syntax and immediately knew what your comment was going to be.